app/views/users/show.html.erb

```diff
@@ -0,0 +1,19 @@
+
+<h1> Mon profil </h1>
+
+<br>
+
+Mes concerts notés 5 étoiles : <br>
+
+<% @user.reviews.where(stars: "5").each do |review| %>
+  <%= review.event.title %> <br>
+  <p> Le <%= review.created_at %> vous avez écrit l'avis suivant :</p>
```

Comment:
Look at the gem rails-i18n for a good base of translations: https://github.com/svenfuchs/rails-i18n
Then look at how i18n works in rails (especially for time purpose): https://guides.rubyonrails.org/i18n.html and https://kitt.lewagon.com/knowledge/tutorials/rails_i18n